views/default/view.php

```diff
@@ -94,3 +96,8 @@
         </div>
     </div>
 </div>
+<script type="text/javascript">
+    if (!window.frameElement) {
```

Comment:
Its not possible, because toolbar.js injected in all pages except `debug` module, so we need detect if module page opened in `iframe`.